package.json

```diff
@@ -13,8 +13,7 @@
   },
   "files": [
     "haunted.js",
-    "index.js",
```

Comment:
don't we want to keep these old bundles in the exposed package, if even in a deprecated state?

Reply:
Maybe... I guess this is the only breaking change. So if we keep this bundle in then it's only a minor release.